narwhal/storage/src/proposer_store.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
```

Comment:
Just yesterday I rewrote all the copyright headers to remove the date so you can change this to 
```suggestion
// Copyright (c) Mysten Labs, Inc.
```